src/App.tsx

```diff
@@ -1,7 +1,37 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string;
+};
+
+export class App extends Component<{}, State> {
+  state = {
+    pressedKey: 'Nothing was pressed yet',
```

Comment:
Your state is about the pressed key. So if there is no pressed key it should be `null` instead of some info message.
And the message you can render while you check the pressed key (see next comment)